eng/packaging.targets

```diff
@@ -22,19 +22,29 @@
                                        '$(IsRIDSpecificProject)' == 'true') and
                                        '$(PreReleaseVersionLabel)' != 'servicing' and
                                        '$(GitHubRepositoryName)' != 'runtimelab'">true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild Condition="'$(BuildAllConfigurations)' != 'true' and
```

Comment:
Nit, I know you are just cherry picking, but I'm just wondering if it would be easier to just automatically set this to false if it wasn't set to true already, as this is the reverse of the previous condition right?

Reply:
Yup, I can do that.